src/main/java/org/spongepowered/common/item/recipe/crafting/DelegateSpongeCraftingRecipe.java

```diff
@@ -0,0 +1,46 @@
+package org.spongepowered.common.item.recipe.crafting;
+
+import com.google.common.base.Preconditions;
+import org.spongepowered.api.item.inventory.ItemStackSnapshot;
+import org.spongepowered.api.item.inventory.type.GridInventory;
+import org.spongepowered.api.item.recipe.crafting.CraftingRecipe;
+import org.spongepowered.api.world.World;
+
+import java.util.List;
+
+public class DelegateSpongeCraftingRecipe extends AbstractSpongeCraftingRecipe {
+
+    private final CraftingRecipe recipe;
+
+    public DelegateSpongeCraftingRecipe(CraftingRecipe recipe) {
+        Preconditions.checkNotNull(recipe, "recipe");
+
+        this.recipe = recipe;
+    }
+
+    @Override
+    public ItemStackSnapshot getExemplaryResult() {
+        return recipe.getExemplaryResult();
```

Comment:
Field accesses should be qualified with `this.` consistently throughout the codebase

Protip: most IDEs can do this automatically